readthedocs/doc_builder/environments.py

```diff
@@ -84,6 +83,8 @@ class BuildCommand(BuildCommandResultMixin):
     :param combine_output: combine stdout/stderr, default=True
     :param input_data: data to pass in on stdin
     :type input_data: str
+    :param str user: User used to execute the command, it can be in form of ``user:group``
```

Comment:
```suggestion
    :param str user: user from inside the container to execute the command, it can be in form of ``user:group``
```

Reply:
This is the base class, not the docker implementation. But we use only the docker implementation, so don't know.

Reply:
Not sure what's best here. Either seems fine to me, given its on the base class.